.travis.yml

```diff
@@ -11,11 +11,15 @@ matrix:
       env: CGO_ENABLED=0
     - go: "1.12.x"
       env: CGO_ENABLED=0
+      env: GO111MODULE=on
     - go: "1.11.x"
       env: CGO_ENABLED=0
+      env: GO111MODULE=on
```

Comment:
As the `go.mod` file specifies that the mini go version is go1.13 we should just use that. 

Let's remove these.

Reply:
I left the env var out of go1.13 because the default behavior of modules is that go will use modules if `go.mod` exists in or above the current directory. Should I still include the env var?